### PR TITLE
Add unsigned bot

### PR DESCRIPTION
**Pull request by pmaslana**
This will create a new bot to find unsigned commits

### CODE REVIEW

File: cmd/notifyUnsignedCommits.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+				prInfo, err := datastore.GetPRData(pr.Repo, int64(pr.PRNumber))
+				if err != nil {
+					slogs.Logr.Error("Error checking PR info in database", "error", err)
+					continue
+				}
+
+				shouldSendMessage := false
+				if prInfo == nil {
+					// New PR, record it and send a message
+					slogs.Logr.Info("Storing data in db")
```
Comment (Starttoaster) [inline cmd/notifyUnsignedCommits.go:69]: Might be worth adding the repo name and PR # to this logline (same goes for other commands in github-bot assuming this same logline exists in those)

Comment (Starttoaster) [reply cmd/notifyUnsignedCommits.go:69]: Actually, imho I would suggest adding repo name and PR # to every logline in this `listPendingPRs` loop

File: internal/github/checkUnsigned.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+					"repository", fullRepo.Name)
+			}
+		}
+	}
+
+	return unsignedPRs, nil
+}
+
+// Checks if a PR has unsigned commits based on the last update from team members
+func isUnsigned(ctx context.Context, githubClient *github.Client, pr *github.PullRequest, teamMembers map[string]bool) (bool, error) {
```
Comment (Starttoaster) [inline internal/github/checkUnsigned.go:70]: Nitpick, but `isUnsigned` sounds more like it's checking an individual commit. `hasUnsignedCommits` sounds more like what it does imo

Comment (pmaslana) [reply internal/github/checkUnsigned.go:70]: fixed

**Review by Starttoaster — APPROVED** [review]

**Pull request merged**